recipes-bsp/imx-atf/imx-atf_2.4.bb

```diff
@@ -67,4 +67,4 @@ do_deploy() {
 addtask deploy after do_compile
 
 PACKAGE_ARCH = "${MACHINE_SOCARCH}"
-COMPATIBLE_MACHINE = "(mx8|imx-boot-container)"
+COMPATIBLE_MACHINE = "(mx8-generic-bsp)"
```

Comment:
As a "wishful thinking": can we pin it to something which is not _generic_? This would allow to build upstream TF-A from _meta-arm_ for `mx8m`, which is sometimes beneficial for layer's users...

I might look into how this can be done.

Reply:
We can, but I believe it would need to be a separated patch, right?

Reply:
Yes, this can be done separate after this PR is merged.